src/Aquarius.ONE.ClientSDK/Operations/Spreadsheet/WorksheetViewConfiguration.cs

```diff
@@ -106,12 +108,7 @@ public class ColumnConfig
         {
             public uint columnWidth { get; set; }
         }
-        [Serializable]
-        public enum EnumHeaderType
-        {
-            column = 1,
-            group = 2
-        }
+      
         public class ColumnHeader

```

Comment:
Can this class be removed also? Along with some other classes in this file.

Reply:
Removed